### PR TITLE
mupen64plus-qt 

### PR DESCRIPTION
**Pull request by Vlad-Romanov**
front-end for mupen64plus

### CODE REVIEW

**Comment by Vlad-Romanov** [comment]
How am I to separate pull request? :( sorry i do not know why it does the other file too....

File: srcpkgs/mupen64plus-qt/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'mupen64plus-qt'
+pkgname=mupen64plus-qt
+version=1.11
+revision=1
+wrksrc="${pkgname}-${version}"
```
Comment (cr6git) [inline srcpkgs/mupen64plus-qt/template]: This is the default, not needed.

**Comment by Vlad-Romanov** [comment]
sorry, I will re do this when I figure out how to better understand...

File: srcpkgs/mupen64plus-qt/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-host-tools"
+makedepends="quazip-devel quazip-qt5-devel qt5-tools-devel"
+depends="quazip quazip-qt5 mupen64plus hicolor-icon-theme"
+short_desc="a front-end for the mupen64plus emulator"
+maintainer="VladTheImpaler <vladimir-romanov-ii@yandex.com>"
+license="BSD"
```
Comment (cr6git) [inline srcpkgs/mupen64plus-qt/template]: Use [SPDX](https://spdx.org/licenses/) standards

File: srcpkgs/mupen64plus-qt/template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+homepage="https://github.com/dh4/mupen64plus-qt"
+distfiles="https://github.com/dh4/${pkgname}/archive/${version}.tar.gz"
+checksum=28a9566f138ff5048dd761fa867892fc7325bdc489a79057360f2e4ddaac8eee
+
+do_install() {
+
+vbin mupen64plus-qt
```
Comment (cr6git) [inline srcpkgs/mupen64plus-qt/template]:
````
do_install() {
	vbin mupen64plus-qt
}
````

**Comment by cr6git** [comment]
@Vlad-Romanov why you close?

**Comment by Vlad-Romanov** [comment]
I was told to do one pull request per package, I cannot make it do only one, it want to do multiple...

**Comment by Vlad-Romanov** [comment]
What I am doing wrong? Is very frustrating.... How to just make it pull request for one package at time?

**Pull request closed without merging**